### PR TITLE
fiat conversion

### PR DESCRIPTION
**Pull request by salomegeo**
## Description
use Number.toFixed instead of BigRational.toDecimal, because toDecimal always rounds it down
819

## Change Type
- Bug Fix

## PR Creator Checklist
- [ ] Code compiles correctly
- [ ] No lint issues exist (verified via `yarn lint`)
- [ ] All unit tests pass (verified via `yarn test`)

## PR Reviewer Checklist
- [ ] Change validated and application spot checked
- [ ] Code styles and best practices met
- [ ] Code is readable and commented when necessary

### CODE REVIEW

File: packages/blockchain-wallet-v4/src/exchange/currency/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export class Currency extends Type {
   toUnit (unit) {
     if (unit && unit.currency === this.currency.code) {
       return Maybe.Just({
-        value: this.value.multiply(BigRational(unit.rate).reciprocate()).toDecimal(unit.decimal_digits),
+        value: Number(this.value.multiply(BigRational(unit.rate).reciprocate())).toFixed(unit.decimal_digits),
```
Comment (plondon) [inline packages/blockchain-wallet-v4/src/exchange/currency/index.js]: `Number(5.00.toFixed(2))`, is this what you want?

Comment (salomegeo) [reply packages/blockchain-wallet-v4/src/exchange/currency/index.js]: Number(5.234).toFixed(2)

Comment (plondon) [reply packages/blockchain-wallet-v4/src/exchange/currency/index.js]: But trailing 0's will still be mitigated by formatFiat, which we're not getting rid of?

Comment (salomegeo) [reply packages/blockchain-wallet-v4/src/exchange/currency/index.js]: This shouldn't affect trailing 0's at all.

I'm not completely sure about trailing 0's in general, where do we want and where do we not want them. If it's consistent we can move that in here too, but if not it should be handled case-by-case bases, which I think is what you did in your pull request.

**Pull request merged**